### ISSUE TITLE
Don't throw Call init form after socket Connect Success RelaySDK(SingnalWair)

### ISSUE THREAD

**Issue by viv578ek** [signalwire/signalwire-node#270]
Hello Team,

     I am not able to throw call after connection Socket success. 
I am making call like this method 
// Make Call 
this.client.newCall({ destinationNumber: this.state.extension, video: { facingMode: 'user' } }) 
// this.state.extension -> 18004337300. USA Country 
No any Log show in terminal after call Init register user.

Error Show : TypeError: undefined is not an object (evaluating '_this4.client.newCall')

Thanks
Vivek Shukla

**Comment by edolix** [comment]
Hi @viv578ek, from the `TypeError` seems the `this.client` is not defined. 

Please make sure to read our [Getting Started](https://docs.signalwire.com/topics/relay-sdk-js/v1/#relay-sdk-for-javascript-getting-started) docs page and follow the [vanilla example here](https://github.com/signalwire/signalwire-node/tree/master/packages/js/examples/vanilla-calling).

**Comment by viv578ek** [comment]
Hi @edolix 
    But I am not getting any log related to call initiations our end.

**Comment by viv578ek** [comment]
And I am also fallow all process as per shared Documents.

**Comment by edolix** [comment]
Since your `this.client` variable is undefined, no call can start so this means no logs. The only log you have is `undefined is not an object`.

Double check your client setup or share a snippet code to see the client init - thanks!

**Comment by viv578ek** [comment]
Ok let me try again

**Comment by edolix** [comment]
I'm going to close this one for now.

Can you open a support ticket from your SignalWire account by visiting `https://<SPACE_NAME>.signalwire.com/support_tickets/new`? You can reference this Github issue in it, but the technical support staff will be able to assist you more and narrow down the issue, since the problem doesn't seem to be a technical SDK one, but something possibly with your configuration / client setup.

Thanks!
